lib/core/test/unit/Cardano/Wallet/Api/Malformed.hs

```diff
@@ -1513,7 +1515,12 @@ instance Malformed (BodyParam (ApiBytesT 'Base64 SerialisedTx))
 -- in Server.hs. Tested by integration tests.
 
 instance Malformed (BodyParam (ApiT SealedTx)) where
-    malformed = [] -- fixme: add a couple things
+    malformed = first BodyParam <$>
+        [ ("", "DecoderErrorDeserialiseFailure 'Shelley Tx' (DeserialiseFailure 0 'end of input')")
+        , ("hello", "DecoderErrorDeserialiseFailure 'Shelley Tx' (DeserialiseFailure 0 'expected list len or indef')")
+        , ("cafecafe", "DecoderErrorDeserialiseFailure 'Shelley Tx' (DeserialiseFailure 0 'expected list len or indef')")
+        , (BSL.fromStrict . T.encodeUtf8 $ validSealedTxBase64, "")
```

Comment:
I would expect the case of using a "validSealedTxBase64" to succeed, but it does not. Fails with:

```
DecoderErrorDeserialiseFailure 'Shelley Tx' (DeserialiseFailure 0 'expected list len or indef')
```

Reply:
Perhaps I'm converting it to ByteString incorrectly.

Reply:
Oops, this endpoint accepts plain bytes, not hex-encoded bytes.

Reply:
Ah yes, you're right. That works!